solidity/contracts/test/TestSendReceiver.sol

```diff
@@ -15,6 +15,8 @@ contract TestSendReceiver is IMessageRecipient {
     using TypeCasts for address;
 
     uint256 public constant HANDLE_GAS_AMOUNT = 50_000;
+    // taken from Message.sol
+    uint256 public constant BODY_OFFSET = 77;
```

Comment:
instead of this, you can import `../libs/Message.sol`, add `using Message for bytes;`, and then do `message.body()`. e.g. https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/test/TestMessage.sol#L24